cmd/raw-segmenter-c/main.c

```diff
@@ -9,14 +10,16 @@ int main(int argc, char *argv[]) {
   if (argc != 2) {
     printf("usage: %s <sqlite.db>\n", argv[0]);
     printf("TABLE should have the following schema:\n");
-    printf("CREATE TABLE extradata(id INTEGER NOT NULL PRIMARY KEY, data "
-           "BLOB);\n");
+
+    printf("CREATE TABLE extradata(id INTEGER NOT NULL PRIMARY KEY, isH264 "
```

Comment:
I ran this through valgrind using https://github.com/nicksanford/video-vcr to produce the db file.